src/posix/client.cpp

```diff
@@ -76,6 +76,27 @@ static void InputCallback(char *aLine)
 }
 #endif // OPENTHREAD_USE_READLINE
 
+static bool FindPrompt(int &aState, char aChar)
+{
+    switch (aChar)
+    {
+    case '>':
+        aState = aState == 0 ? 1 : -1;
```

Comment:
define state enum type?

Reply:
I tend to enhance this later because this needs a rewrite of multiple functions which is beyond the purpose of this PR.

Reply:
I am fine to enhance later.